<launch>
    <node name="joy_node" pkg="joy" type="joy_node" />
    <node name="joystick_remap" pkg="rc_vip" type="Joystick.py" />
</launch>
